### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_ColorContrast:The color radio buttons are below the ratio 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9203]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_SQLAzureDataStudioBenchmark;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.4.11;#A11y_SQlADS_WCAG2.1;#ColorContrast;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tools:Accessibility insights for desktop
MAS References:MA1.4.11

**Pre-requisite: apply the various themes available for the application.**

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Server Group' icon.
4. Launch accessibility insights for desktop and select the button "Contrast"

**Actual:**
On checking the color contrast of the color button on the selection the color is below 3:1 value.

**Issue is observed on every theme.**

**Expected:**
The ratio for all colors in every theme should be 3:1 or greater.

**User Impact:**
Low vision users will face difficulty in viewing the buttons.

**Attachment for Reference :**
![image](https://user-images.githubusercontent.com/24580570/74803592-76b80480-5303-11ea-9b96-c5b9e7fd8980.png)

**Comment by alanrenmsft** [comment]
this is by design, user can pick the colors they feel comfortable with. we have gone through this review before.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by Yashi1201** [comment]
@alanrenmsft Could you pls reopen the bug as the issue is still repro.
Version: 1.42.0-insider(user setup)
![9203_issue repro](https://user-images.githubusercontent.com/93374043/217498355-ab3c8a78-ea3c-4f9d-b21c-ef4b8833cf8d.png)

**GithubTags**:#Regressed:02-08-23;

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.42.0
![image](https://user-images.githubusercontent.com/85863999/218070806-a07f8b70-f254-44f0-a7e5-d6d28886c0f7.png)

**Comment by Roopesh-T** [comment]
@alanrenmsft I agree that the user can pick the colors, but if there are light colors provided in the list of color choice, the low vision users will have a smaller number of options they can go with. If all the colors with proper ratios, the users can utilize all the options from the list provided.

**Git Hub Tag**:#Regressed:02-15-23;

**Comment by alanrenmsft** [comment]
@Roopesh-T I've fixed the issue, please use the latest insider build to verify it.

![image](https://user-images.githubusercontent.com/13777222/219503578-5c6ed948-484d-40f3-b57f-009f97385134.png)

**Comment by sspatil123** [comment]
Git Hub Tag:#Regressed:02-17-23;#Closed;Version 1.42.0 insider As issue is fixed in above environment you can close that bug.
![image](https://user-images.githubusercontent.com/85863999/219541871-b8170669-7c23-4669-a426-0317df5cc764.png)
![image](https://user-images.githubusercontent.com/85863999/219541896-e5cc2bff-be75-4366-8acd-d5921c7d8bcf.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
